CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -185,7 +185,7 @@ public function setDefaultValues() {
     }
 
     if ($this->_refund) {
-      $defaults['total_amount'] = abs($this->_refund);
+      $defaults['total_amount'] = number_format(abs($this->_refund), 2);
     }
```

Comment:
@eileenmcnaughton so for now, should we allow the current fix that is formatting refund money up to 2 decimal place?